build.gradle

```diff
@@ -32,6 +32,17 @@ repositories {
             includeGroup "maven.modrinth"
         }
     }
+    exclusiveContent {
+        forRepository {
+            maven {
+                name = "mschae23"
+                url = "https://mschae23.de/git/api/packages/mschae23/maven"
```

Comment:
While there is nothing wrong with this, I would prefer not to add everyones maven here. I see you also have this build on modrinth can we just pull it from there instead?

Reply:
I wasn't sure and Modrinth's docs said to prefer authors' own repositories. I'll change it to use that.

Reply:
I think this project is a bit of a special case. The reason they will suggest that is they are unable to provide the full metadata in the POM, so depedencies will not be correct. But as this project purely needs the jar to include it in a pack thats not a problem for us.